### PR TITLE
[MB-2493] Add image file support to orders document viewer

### PR DESCRIPTION
**Pull request by duncan-truss**
## Description

This extends the document viewer to support image file (jp(e)g, gif, and png) formats in addition to pdf files.  The library we're using @trussworks/react-file-viewer has a [corresponding PR](https://github.com/trussworks/react-file-viewer/pull/3) that makes the changes there to enable image support.

## Reviewer Notes

One thing to notice are how images are scaled to fit in the document viewer view.  This is how the library works out of the box.  I didn't touch these calculations, but with the header overlay and top padding this will push a portion of the image out of the viewport and requires scrolling if it is larger in height than width.  We may want to change this to display larger images (maybe only scaled to fit in the width dimension) seeing as the zoom controls are not present for images the way they are for pdfs.

Document Viewer tests are skipped in loki because of flakiness loading the large files.  I could see if we can reenable them for the image stories only and not the pdf story.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

1. Log in as the TOO user
2. Navigate to a move orders URL [http://officelocal:3000/moves/1/orders](http://officelocal:3000/moves/1/orders)
3. Open the menu to see that there is an image file included with the pdf file and can be switched to

### Storybook
[Document Viewer Component Image Viewer](http://localhost:6006/iframe.html?id=components-document-viewer-document-viewer--image-viewer)

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-2493) for this change

## Screenshots

Screen Shot 2020-09-04 at 3 18 10 PM

### CODE REVIEW

File: src/pages/Office/MoveOrders/MoveOrders.jsx [hunk 1]
```diff
@@ -24,6 +25,11 @@ const MoveOrders = () => {
       fileType: 'pdf',
       filePath: samplePDF3,
     },
+    {
+      filename: 'Test File 4.jpg',
+      fileType: 'jpg',
+      filePath: sampleJPG,
+    },
```
Comment (monfresh) [inline src/pages/Office/MoveOrders/MoveOrders.jsx:32]: Do we want to add the PNG and GIF files here too, or is the JPG enough?

**Comment by monfresh** [comment]
This looks good to me. I noticed a layout issue that I didn't see in your screenshot. I'm not sure if it's related to this PR or not. Notice the "View Orders" in the top right in this screenshot:

Screen Shot 2020-09-04 at 5 00 07 PM

File: src/pages/Office/MoveOrders/MoveOrders.jsx (hunk 1 above)
Comment (duncan-truss) [reply src/pages/Office/MoveOrders/MoveOrders.jsx:32]: I included them all in the storybook version, tests, and added them to the react-file-viewer library.  The hardcoded files here are likely getting replaced immediately in https://dp3.atlassian.net/browse/MB-3243

**Comment by duncan-truss** [comment]
> This looks good to me. I noticed a layout issue that I didn't see in your screenshot. I'm not sure if it's related to this PR or not. Notice the "View Orders" in the top right in this screenshot:
> 
> Screen Shot 2020-09-04 at 5 00 07 PM

That is a known issue because that view orders form has been built yet.  It's the same component used on the payment request service item review where you can see the sidebar built out with the document viewer.

**Review by brooksr4 — APPROVED** [review]
It appears to be working correctly

**Review by suzubara — APPROVED** [review]
awesome!

**Pull request merged**